### ISSUE TITLE
left bundle branch block/hemiblock

### ISSUE THREAD

**Issue by kanems** [monarch-initiative/mondo#2536]
**Mondo term (ID and Label):**
MONDO:0000991 
left bundle branch block

**Xref that should be fixed (ID and label):**
SCTID:4973001 (MONDO:equivalentTo)
UMLS:C0155702 (MONDO:equivalentTo)
ICD9:426.2 (i2s)
ICD10:I44.60 (DOID:10272)
 
All of these Xrefs map to "left bunch branch hemiblock," per UMLS. Within the UMLS hierarchy, this "hemiblock" term is a child of : 
"left bunch branch block"  which has the following identifiers: UMLS:C0023211; SCTID: 63467002;  ICD10:I44.7 and HPO: HP:0011713

So to keep this term/concept as labeled, update to the second grouping of cross-refs. Consider creating a new term for left bunch branch hemiblock with the first grouping of X-refs.

**Your nano-attribution (ORCID)**
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/)

**Other comments:**

**Comment by nicolevasilevsky** [comment]
@pnrobinson I wonder if these term should be in HPO and not in Mondo?

Can you give your thoughts?

see:
https://www.ebi.ac.uk/ols/ontologies/mondo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMONDO_0020803

MONDO_0007241 bundle branch block, familial isolated complete right xrefs OMIM:113950, so this seems like an appropriate disease term, but I'm not sure about the other terms.

Thanks for your input!

**Comment by pnrobinson** [comment]
LBB is a phenotypic feature and not a disease. It already is in HPO
https://hpo.jax.org/app/browse/term/HP:0011713

I agree that MONDO_0007241 bundle branch block, familial isolated complete right xrefs OMIM:113950 is a good MONDO term.

**Comment by nicolevasilevsky** [comment]
**proposal to obsolete:**
MONDO_0020803 'bundle branch block', consider HP:0011710 Bundle branch block
MONDO_0000991 'left bundle branch block', consider HP:0011713 Left bundle branch block
MONDO_0001662 'right bundle branch block (disease)', consider HP:0011712 Right bundle branch block
MONDO_0020755 'heart block', consider HP:0012722 Heart block
